### PR TITLE
Fix comments

### PR DESCRIPTION
**Pull request by habemuscode**
Fix the following comments:

- https://github.com/grammyjs/website/pull/442#pullrequestreview-1195844440
- https://github.com/grammyjs/website/pull/555#discussion_r1033581869
- https://github.com/grammyjs/website/pull/505#discussion_r1033687861
- https://github.com/grammyjs/website/pull/505#discussion_r1033688382
- https://github.com/grammyjs/website/pull/505#discussion_r1033689138

### CODE REVIEW

**Comment by roj1512** [comment]
Undraft this?

**Comment by habemuscode** [comment]
@roj1512 I will do when receive answer in the group to include extra fix or not

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

File: site/docs/es/hosting/fly.md [hunk 1]
```diff
@@ -301,8 +301,7 @@ kill_timeout = 5
 [processes]
   app = "run --allow-net ./app.ts"
 
-# Simply omitting the whole [[services]] section 
-# since we are not listening to HTTP
+# Simplemente omitiendo toda la sección de [[servicios]] ya que no estamos escuchando HTTP.
```
Comment (CikiMomogi) [inline site/docs/es/hosting/fly.md:304]:
```suggestion
# Simplemente omitiendo toda la sección de [[services]] ya que no estamos escuchando HTTP.
```

**Review by CikiMomogi — CHANGES_REQUESTED** [review]

**Review by roj1512 — APPROVED** [review]

**Review by CikiMomogi — APPROVED** [review]

**Pull request merged**